tests/integration/test_store.py

```diff
@@ -1845,6 +1845,7 @@ def test_span_extraction_with_metrics_summary(
     project_config = mini_sentry.add_full_project_config(project_id)
     project_config["config"]["features"] = [
         "organizations:custom-metrics",
+        "projects:span-metrics-extraction",
```

Comment:
Bonus points for a test that validates that no feature flag results in no metrics.

Reply:
Bonus point unlocked.